boot/zephyr/Kconfig

```diff
@@ -99,6 +111,12 @@ config BOOT_ECDSA_TINYCRYPT
 	bool "Use tinycrypt"
 	select BOOT_USE_TINYCRYPT
 
+config BOOT_ECDSA_X509
+bool "USE X509 with X509 signature"
+	select BOOT_X509
+	select BOOT_USE_MBEDTLS
```

Comment:
In Application side, We enabled MBEDTLS at prj.conf file
Can we follow same for bootloader

Reply:
Mbedtls needs to be enabled only for the X509 and not for Key based approach and usually we keep only one macro open for user